rotkehlchen/history/events/structures/base.py

```diff
@@ -244,16 +245,16 @@ def serialize(self) -> dict[str, Any]:
             'notes': self.notes,
         }
 
-    def serialize_for_csv(self) -> dict[str, Any]:
+    def serialize_for_csv(self, fiat_value: FVal) -> dict[str, Any]:
         """Serialize event data for CSV export"""
         entry = self.serialize()
         balance = entry.pop('balance')
         new_dict = {}
         for key, value in entry.items():
             new_dict[key] = value
-            if key == 'asset':  # input the amount/usd value right after the asset
+            if key == 'asset':  # input the amount/fiat value right after the asset
```

Comment:
mention here that we don't use the usd_value here because we sport the value in the user selected currency